release/api/v1alpha1/artifact_types_test.go

```diff
@@ -121,3 +121,140 @@ func TestImageTag(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
I loathe table tests, so I am willing to convert these, but this is the style used here.